src/EntryPoint/RestEntryPoints.ts

```diff
@@ -57,4 +61,19 @@ export class RestEntryPoints implements EntryPoints {
 
     return res.data.items;
   }
+
+  public async getEntryPointDetails(
+    projectId: string,
+    entryPointId: string
+  ): Promise<GetEntryPointDetailsResponse> {
+    try {
+      const res = await this.client.get<GetEntryPointDetailsResponse>(
+        `api/v2/projects/${projectId}/entry-points/${entryPointId}`
+      );
+
+      return res.data;
+    } catch (_) {
+      return undefined;
```

Comment:
same thing, there could be other errors like 429 too many requests that do not indicate absence of entrypoint